pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/HierarchyTopicPolicies.java

```diff
@@ -60,6 +60,7 @@ public class HierarchyTopicPolicies {
 
     final PolicyHierarchyValue<Boolean> schemaValidationEnforced;
     final PolicyHierarchyValue<EntryFilters> entryFilters;
+    final PolicyHierarchyValue<AutoSubscriptionCreationOverride> autoSubscriptionCreationOverride;
```

Comment:
Can we just use `PolicyHierarchyValue<Boolean> autoSubscriptionCreation;` here?

Reply:
I have removed variable  `autoSubscriptionCreationOverride` because we are not using it in `HierarchyTopicPolicies`.